CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+v2.6.2 / 2024-09-25
+========================
+* Cherry-pick #552 ([#552](https://github.com/openebs/zfs-localpv/pull/552),[@Abhinandan-Purkait](https://github.com/Abhinandan-Purkait))
```

Comment:
These entries should communicate changes made in the releases. PR-centric language doesn't help here.
```suggestion
* fix(chart): handle trailing slash (/) in csi plugin kubelet directory ([#532])(https://github.com/openebs/zfs-localpv/pull/532), [@w3aman](https://github.com/w3aman))
* fix(chart): remove anti-affinity from csi controller ([#552](https://github.com/openebs/zfs-localpv/pull/552),[@Abhinandan-Purkait](https://github.com/Abhinandan-Purkait))
```

The cherry-pick PR is a container for these changes to conveniently move to the release branch, however the PR itself is not an isolated change. The individual change instances should be documented.

Reply:
Sure.